src/interface/src/app/region-selection/region-selection.component.html

```diff
@@ -0,0 +1,19 @@
+<div class="center-block">
+  <section class="image-section">
+    <img src="assets/svg/planscape-image.svg">
+  </section>
+  <section class="content-section">
+    Different regions have varied datasets available.
```

Comment:
Suggested rewording: "Different regions are associated with different datasets from the Regional Resource Kits."

Reply:
Thanks for reviewing! I pulled the text from the mocks- I'll double check with UX?

Reply:
Checked with Alex and the wording will probably change once we get a UX Writer/Technical writer. Updated the text to your suggestion.